docs/DeveloperGuide.md

```diff
@@ -173,6 +173,96 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+
+
+### The Person class
+
+ManageHR keeps track of employees within the company with the use of `Person` and `UniquePersonList`. The `UniquePersonList` serves as a container for the `Person` objects,
+while enforcing the constraints that no 2 employees have the same name.
+
+The `Person` class contains the following attributes.
+<puml src="diagrams/PersonObjectDiagram.puml", width="250"></puml>
```

Comment:
Could you set the diagram width to something larger for better readability e.g. 550? Same for the other UML diagrams